apps/gateway/lib/gateway/auth/token_manager.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Gateway.Auth.TokenManager do
+  @moduledoc """
+  Module responsible to verify and validate the JWT emitted by gateway app.
```

Comment:
Can we update this? I think this documentation is for `Arena.Authentication.GatewayToken`